autonomy/fsm/scaffold/generators/components.py

```diff
@@ -70,11 +70,25 @@ def get_file_content(self) -> str:
             COPYRIGHT_HEADER,
             ROUNDS.HEADER.format(**self.template_kwargs),
             self._get_rounds_section(),
-            self.ABCI_APP_CLS.format(**self.template_kwargs),
+            self.ABCI_APP_CLS.format(
+                **self.template_kwargs,
+                db_pre_conditions=self._get_pre_condition_rounds_str(),
+                db_post_conditions=self._get_post_condition_rounds_str(),
+            ),
         ]
 
         return "\n".join(file_content)
 
+    def _get_pre_condition_rounds_str(self) -> str:
+        """Get string with pre_conditions rounds."""
+        return _indent_wrapper("\n".join([f"\t{round}: []," for round in self.rounds]))
+
+    def _get_post_condition_rounds_str(self) -> str:
+        """Get string with pre_conditions rounds."""
+        return _indent_wrapper(
+            "\n".join([f"\t{round}: []," for round in self.degenerate_rounds])
```

Comment:
We need this for the terminal rounds instead.

Reply:
done